templates/packages/partner_search.html

```diff
@@ -1,56 +1,101 @@
 {%extends 'base.html' %}
 
 {% block title%}
-    תוצאות חיפוש
+תוצאות חיפוש
 {%endblock%}
 {% block content%}
-    <script>
-        window.onload = function() {
-            reveal_search();
-        }
-    </script>
-    <div class="container">
-        <div class="row justify-content-center mb-3">
-            <div class="col text-center">
-                {% include "packages/parts/search.html" %}
+<div class="container-xxl mb-5">
+    <form method="get" dir="ltr" onkeydown="return event.key != 'Enter';">
+        <div class="row">
+            <div class="col">
+                <div class="input-group mb-3">
+                    <!-- This part is not designed for mobile really... So the formatting on the narrowest devices is bad -->
+                    <span class="input-group-text">Origin</span>
+                    <input id="origin-address" name="origin-address" class="col form-control"
+                        placeholder="Street 12, City" autocomplete="off" />
+                    <input id="origin-city" name="origin-city" type="hidden" />

```

Comment:
Why would any of this search work? The user is `origin-address` and in the server you are reading from the hidden fields here which are untouched....

Reply:
I did test it, while backmerging the branch with dev I forgot to include the autocomplete script on this page...